packages/components/src/components/tooltip/tooltip.css

```diff
@@ -19,6 +19,7 @@
   --line-height: var(--telekom-typography-line-spacing-standard);
   --spacing: var(--telekom-spacing-composition-space-02)
     var(--telekom-spacing-composition-space-04);
+  --width: 'min-content';
```

Comment:
This is no taking effect because of the quotes. It should be `min-content` not `'min-content'`, no?

If you make it `min-content` it breaks as before. I think we should make this `--width: auto` and then people facing the Modal use-case can set it to `--width: min-content`. Wdyt?

Reply:
oh that was my bad, and makes sense to set it to auto 🙏